react/components/molecules/image-carrousel/image-carrousel.js

```diff
@@ -0,0 +1,185 @@
+import React, { PureComponent } from "react";
+import {
+    Dimensions,
+    Image,
+    Modal,
+    ScrollView,
+    StyleSheet,
+    Text,
+    View,
+    ViewPropTypes
+} from "react-native";
+import PropTypes from "prop-types";
+
+import { isTabletSize } from "../../../util";
+
+import { ButtonIcon } from "../../atoms";
+
+export class ImageCarrousel extends PureComponent {
+    static get propTypes() {
+        return {
+            selectedImage: PropTypes.number,
+            resizeModeFullScreen: PropTypes.string,
+            images: PropTypes.array,
+            onVisible: PropTypes.func,
+            style: ViewPropTypes.style
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            selectedImage: 0,
+            resizeModeFullScreen: undefined,
+            images: [],
+            onVisible: () => {},
+            style: {}
+        };
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.scrollCoefficient = Dimensions.get("window").width;
+
+        this.state = {
+            visible: false,
+            selectedImageData: this.props.selectedImage

```

Comment:
In react we're not using `*Data` (no need because you need to access it via `this.props.selectedImage` and `this.state.selectedImage`

```suggestion
            selectedImage: this.props.selectedImage
```